packages/compass-collection/src/modules/collection-tab.ts

```diff
@@ -607,6 +611,63 @@ export const analyzeCollectionSchema = (): CollectionThunkAction<
   };
 };
 
+const validateFakerSchema = async (
+  fakerSchema: MockDataSchemaResponse,
+  logger: Logger
+) => {
+  const { faker } = await import('@faker-js/faker');
```

Comment:
Due to some restrictions around how compass-web is packaged for consumption, we can't have dynamic imports in compass without some extra setup, essentially they need to be inside the published bundle. When testing this code inside mms it breaks on this import for me. Did it work for you?

The easiest thing right now would be to just make it into a static import at the top of the file (you can also add faker to the list of dependencies [here](https://github.com/mongodb-js/compass/blob/d8a0d16731f94d7f7c78c961d1aeb48544fb2e68/packages/compass-web/webpack.config.js#L324-L335) to make sure that we are not crossing the 10mb limit for compass-web bundle)